tests/Backend/CommonPart1/AlterTableTest.php

```diff
@@ -204,7 +204,38 @@ public function testTooManyColumns(): void
                 new CsvFile($importFile),
                 []
             );
-            $this->fail('There were 5000 columns man. fail.');
+            $this->fail('There were 5000 columns which should fail.');
+        } catch (ClientException $e) {
+            $this->assertEquals('storage.tables.validation.tooManyColumns', $e->getStringCode());
+        }
+    }
+
+    public function testTooManyColumnsCountFailed(): void
+    {
+        $this->skipTestForBackend([
+            self::BACKEND_EXASOL,
+            self::BACKEND_BIGQUERY,
+        ], 'Backend has no limit regarding number of columns');
+
+        $data = [
+            'name' => 'tooManyColumns',
+            'primaryKeysNames' => [],
+            'columns' => [],
+        ];
+        // generate table definition with too many columns - over limit on every backend
+        // don't make that number ridiculously big. RequestObject won't make it...
+        for ($i = 1; $i <= 2100; $i++) {
+            $data['columns'][] = [
+                'name' => 'col' . $i,
+                'basetype' => 'BOOLEAN',
+            ];
+        }
+        try {
+            $this->_client->createTableDefinition(
```

Comment:
nerikal jsi, ze toto presunes nekam jinam, kde maji projekty (i synapse nebo snflk) tables-definition feature? poznas to podle init commandu v connection

Reply:
jj, pridal jsem komentar puvodne do Connection PR, ted uz je tady...